src/pages/bubblegum-v2/transfer-cnfts.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Transferring Compressed NFTs
+metaTitle: Transferring Compressed NFTs | Bubblegum v2
+description: Learn how to transfer compressed NFTs on Bubblegum
+---
+
+The **transferV2** instruction can be used to transfer a Compressed NFT from one owner to another. To authorize the transfer, either the current owner or the delegate authority — if any — must sign the transaction.
+
+Note that this instruction updates the Compressed NFT and therefore replaces the leaf on the Bubblegum Tree. This means additional parameters must be provided to verify the integrity of the Compressed NFT. Since these parameters are common to all instructions that mutate leaves, they are documented [in the following FAQ](/bubblegum-v2/faq#replace-leaf-instruction-arguments). Fortunately, we can use a helper method that will automatically fetch these parameters for us using the Metaplex DAS API.
+
+{% callout title="Transaction size" type="note" %}
+If you encounter transaction size errors, consider using `{ truncateCanopy: true }` with `getAssetWithProof`. See the [FAQ](/bubblegum-v2/faq#replace-leaf-instruction-arguments) for details.
+{% /callout %}
+
+## Transfer a Bubblegum V2 Compressed NFT
+
+The instruction accepts the following parameters:
```

Comment:
Also collection must be passed in if the item is in a collection, so that collection plugins like `Royalties` can be checked

Reply:
Nit: do you point out anywhere that collection must be added for items that are in a collection?

Reply:
Only had it in the code example, not in the parameter list. Added.